Tools/PowershellModule/doc/PowerShell/Get-WinGetManifest.md

```diff
@@ -46,35 +41,35 @@ The following Azure Modules are used by this script:
 Get-WinGetManifest -Path "C:\AppManifests\Microsoft.PowerToys"
 ```
 
-Returns an array of all Manifest objects based on the files found within the specified Path.
+Returns an array of the Application Manifest objects based on the files (*.yaml or *.json) found within the specified Path.
 
 ### EXAMPLE 2
 ```
 Get-WinGetManifest -Path "C:\AppManifests\Microsoft.PowerToys\Microsoft.PowerToys.json"
 ```
 
-Returns a Manifest object (*.json) of the specified JSON file.
+Returns an Application Manifest object (*.json) of the specified JSON file.
 
 ### EXAMPLE 3
 ```
 Get-WinGetManifest -FunctionName "PrivateSource" -ManifestIdentifier "Windows.PowerToys"
 ```
 
-Returns a Manifest object of the specified Manifest Identifier that is queried against in the REST APIs.
+Returns an Manifest object of the specified Application Manifest Identifier that is queried against in the REST APIs.
 
 ### EXAMPLE 4
 ```
 Get-WinGetManifest -FunctionName "PrivateSource" -ManifestIdentifier "Windows.PowerToys" -SubscriptionName "Visual Studio Subscription"
 ```
 
-Returns a Manifest object of the specified Manifest Identifier that is queried against in the REST APIs from the specified Subscription Name.
+Returns an Application Manifest object of the specified Manifest Identifier that is queried against in the REST APIs from the specified Subscription Name.
```

Comment:
"queries against in"?

Lets match terminology with the specific rest apis.
'returned by the ManifestGet REST api for the specified  package identifier"...etc

Reply:
This cmdlet doesn't just source it's content from the REST API.

Reply:
The commandlet supports other things, but this example is talking about pulling from the rest api.  Or the example text is bogus to begin with?